app/javascript/components/Results/Jobs/Jobs.tsx

```diff
@@ -68,26 +70,33 @@ export const Jobs = ({ jobs=[] }: JobsProps) => {
     moreJobs = jobs.slice(MAX_JOBS_IN_COLLAPSE_VIEW, jobs.length);
   }   
 
+  const jobOpeningsHeader = (agency: string | undefined) => {
+    if (agency) {
+      return `${i18n.t('jobOpenings')} ${i18n.t('atAgency', { agency })}`;
+    }
+    return i18n.t('federalJobOpenings');
+  };
+
   return (
     <>
-      {jobs?.length > 0 && (
-        <StyledWrapper styles={styles}>
-          <div className='search-item-wrapper search-jobs-item-wrapper'>
-            <GridContainer className='jobs-title-wrapper'>
-              <Grid row gap="md">
-                <Grid col={true}>
-                  <h2 className='jobs-title-wrapper-label'>
-                    {i18n.t('federalJobOpenings')}
-                  </h2>
-                </Grid>
-                <Grid col={true} className='jobs-logo-wrapper'>
-                  <a className="usajobs-logo" href="https://www.usajobs.gov/">
-                    <img alt="USAJobs.gov" />
-                  </a>
-                </Grid>
+      <StyledWrapper styles={styles}>
+        <div className='search-item-wrapper search-jobs-item-wrapper'>
+          <GridContainer className='jobs-title-wrapper'>
+            <Grid row gap="md">
+              <Grid col={true}>
+                <h2 className='jobs-title-wrapper-label'>
+                  {jobOpeningsHeader(agencyName)}
+                </h2>
               </Grid>
-            </GridContainer>
-
+              <Grid col={true} className='jobs-logo-wrapper'>
+                <a className="usajobs-logo" href="https://www.usajobs.gov/">
+                  <img alt="USAJobs.gov" />
+                </a>
+              </Grid>
+            </Grid>
+          </GridContainer>
+          
+          {jobs?.length > 0 ? <>
```

Comment:
Do we need this empty element?

Reply:
We did it because multiple child elements need a parent. Let me know if there is a way to get rid of them. I didn't want an unnecessary div tag as a parent wrapper.

Reply:
I didn't see the multiple child